buildenv/jenkins/variables/defaults.yml

```diff
@@ -128,3 +128,29 @@ aix_ppc-64_cmprssptrs:
     8: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-ccache --with-jobs=8'
     9: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
     10: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
+#========================================#
+# Windows x86 64bits Compressed Pointers
+#========================================#
+win_x86-64_cmprssptrs:
+  boot_jdk:
+    8: '/cygdrive/c/openjdk/jdk7'
+    9: '/cygdrive/c/openjdk/jdk8'
+    10: '/cygdrive/c/openjdk/jdk9'
+  release:
+    8: 'windows-x86_64-normal-server-release'
+    9: 'windows-x86_64-normal-server-release'
+    10: 'windows-x86_64-normal-server-release'
+  freemarker: '/cygdrive/c/openjdk/freemarker.jar'
+  extra_configure_options:
+    8: '--with-freetype-include=/cygdrive/c/openjdk/freetype-2.5.3/include --with-freetype-lib=/cygdrive/c/openjdk/freetype-2.5.3/lib64 --disable-ccache'
+    9: '--with-freetype-src=/cygdrive/c/openjdk/freetype-2.5.3 --with-toolchain-version=2013 --disable-ccache'
+    10: '--with-freetype-src=/cygdrive/c/openjdk/freetype-2.5.3 --with-toolchain-version=2013 --disable-ccache'
```

Comment:
why `--disable-ccache`?

Reply:
We don't have ccache installed on Windows. Adopt also has this option for their Windows builds.